recipes/nnetsauce/meta.yml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nnetsauce" %}
+{% set version = "0.8.3" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/Techtonique/{{ name }}/tree/master/dist/{{ name }}-{{ version }}.tar.gz
+    sha256: 2163e76d6edc6339207b8ee89275ab0fc9c8846f1f6a6997c9fed5628ad5ae0e
+
+build:  
+    number: 0  
```

Comment:
It looks like this package requires python >=3.5 so please add:

`skip: true  # [py<35]`

in the `build` section here.

Reply:
fixed